tools/pika_migrate/format_code.sh

```
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+find include src tools -regex '.*\.\(cpp\|hpp\|c\|h\|cc\)' | xargs clang-format -i
```

Shellcheck (0.11.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)